lifecycle/services/operation.go

```diff
@@ -35,7 +36,13 @@ type ReleaseBundleOperation interface {
 	getSigningKeyName() string
 }
 
-func (rbs *ReleaseBundlesService) doOperation(operation ReleaseBundleOperation) ([]byte, error) {
+type operationParams struct {
+	httpMethod  string
+	contentType string
+}
+
+func (rbs *ReleaseBundlesService) doOperation(operation ReleaseBundleOperation, operationParams ...operationParams) (body []byte, err error) {
```

Comment:
The intention here to reuse code is great, but I think it will be simpler and more self-explanatory if we create two functions, each for every http method. Both can use a third function with a common code.